setup.py

```diff
@@ -22,13 +22,13 @@
 
 project_name = 'jax'
 
-_current_jaxlib_version = '0.4.19'
+_current_jaxlib_version = '0.4.20'
 # The following should be updated with each new jaxlib release.
 _latest_jaxlib_version_on_pypi = '0.4.19'
 _available_cuda11_cudnn_versions = ['86']
 _default_cuda11_cudnn_version = '86'
 _default_cuda12_cudnn_version = '89'
-_libtpu_version = '0.1.dev20231018'
+_libtpu_version = '0.1.dev20231030'
```

Comment:
Change this if you are not releasing today

Reply:
11